net-kit/autogen.kit.d/base.yml

```diff
@@ -2627,4 +2627,159 @@ gitlab_freedesktop_dirlisting:
               rmdir "${ED}"/var{/lib{/NetworkManager,},} || die
             }
 
+libcupsfilters:
+  generator: builtin-github
+  template:
+    engine: helm
+
+  defaults:
+    files_dir: "{{ .Values.pn }}"
+    template: ../../templates/simple.tmpl
+    category: net-print
+    github:
+      user: OpenPrinting
+      repo: libcupsfilters
+    assets:
+      - name: '{{ .Values.pn }}-{{ .Values.version }}.tar.xz'
+        url: 'https://github.com/OpenPrinting/{{ .Values.pn }}/releases/download/{{ .Values.version }}/{{ .Values.pn }}-{{ .Values.version }}.tar.xz'
```

Comment:
Cosmetic: When there assets and `builtin-github` you can also just use `matcher` of the tarball instead of the full path. See for example: https://github.com/macaroni-os/kit-fixups/blob/mark-unstable/core-server-kit/autogen.kit.d/telemetry.yml#L17.

It's fine this way too. Just sharing an alternative way for github packages.